### PR TITLE
fix: guest label not aligned [AR-2178]

### PR DESCRIPTION
**Pull request by MohammedMokresh**
<table>
<td>
  <a href="https://wearezeta.atlassian.net/browse/AR-2178" title="AR-2178" target="_blank">BugAR-2178</a>  Guest, External etc labels not alligned
  </td></table>
  <br />
 

----
#### PR Submission Checklist for internal contributors

- The **PR Title**
  - [x] conforms to the style of semantic commits messages¹ supported in Wire's Github Workflow²
  - [x] contains a reference JIRA issue number like `SQPIT-764`
  - [x] answers the question: _If merged, this PR will: ..._ ³

- The **PR Description**
  - [ ] is free of optional paragraphs and you have filled the relevant parts to the best of your ability
----

# What's new in this PR?

### Issues

guest label was not aligned with the name in the search list 

### Causes (Optional)

we were passing fixed padding to the label 

### Solutions

make the user badge accept top padding argument so it we can pass different padding for different occurences 

### Attachments (Optional)
Screenshot 2022-08-22 at 13 37 14
Screenshot 2022-08-22 at 13 39 37

----
#### PR Post Submission Checklist for internal contributors (Optional)

 - [ ] Wire's Github Workflow has automatically linked the PR to a JIRA issue
----
#### PR Post Merge Checklist for internal contributors

 - [ ] If any soft of configuration variable was introduced by this PR, it has been added to the relevant documents and the CI jobs have been updated.
----
##### References
1. https://sparkbox.com/foundry/semantic_commit_messages
1. https://github.com/wireapp/.github#usage
1. E.g. `feat(conversation-list): Sort conversations by most emojis in the title #SQPIT-764`.

### CODE REVIEW

File: app/src/main/kotlin/com/wire/android/ui/common/UserBadge.kt [hunk 1]
```diff
@@ -26,12 +25,16 @@ import com.wire.kalium.logic.data.user.ConnectionState
  * @param startPadding - Dp of labels left padding
  */
 @Composable
-fun UserBadge(membership: Membership, connectionState: ConnectionState? = null, startPadding: Dp = 0.dp) {
+fun UserBadge(
+    membership: Membership, connectionState: ConnectionState? = null,
```
Comment (borichellow) [inline app/src/main/kotlin/com/wire/android/ui/common/UserBadge.kt:29]: can you please move `connectionState` to the new line

**Review by borichellow — COMMENTED** [review]
good catch! Just a small code-style comment  :)

**Review by borichellow — APPROVED** [review]

**Comment by AndroidBob** [comment]
Build [2](https://10.10.124.134/job/wire-android-reloaded/job/PR-881/2/) **succeeded**.

The build produced the following APK's:
- [com.wire.android-vv0.0.1(4220243)(4220243)-dev-release.apk](https://z-lohika.s3-eu-west-1.amazonaws.com/megazord/android/reloaded/dev/release/com.wire.android-vv0.0.1(4220243)(4220243)-dev-release.apk)

**Comment by AndroidBob** [comment]
Build [3](https://10.10.124.134/job/wire-android-reloaded/job/PR-881/3/) **succeeded**.

The build produced the following APK's:
- [com.wire.android-vv0.0.1(4220334)(4220334)-dev-release.apk](https://z-lohika.s3-eu-west-1.amazonaws.com/megazord/android/reloaded/dev/release/com.wire.android-vv0.0.1(4220334)(4220334)-dev-release.apk)

**Pull request merged**